dist/init/systemd/rkt-api-tcp.socket

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=rkt api service socket
+PartOf=rkt-api.service
+
+[Socket]
+ListenStream=15441
```

Comment:
@alban To be honest, I'm not sure if it works. I would think the code you referenced has to change to accept a fd to in order to support socket activation, but that reflects very little low-level knowledge. This PR was sort of inquisitive/WIP.